src/pages/ViewInterviewPage.tsx

```diff
@@ -85,8 +93,25 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
       borderRadius="lg"
       p={6}
     >
+      <Modal
+        isCentered
+        motionPreset="slideInBottom"
+        isOpen={isOpen}
+        size={"xl"}
+        onClose={onClose}
+      >
+        <ModalOverlay />
+        <ModalContent maxH="800px" maxW="700px">
+          <ModalCloseButton />
+          <ReportPopUp
+            associated_id={interview.interviewId}
+            reporter_user_id={member?.id || -1}
```

Comment:
should we use -1 here or is undefined fine? if `member?.id` is undefined this will just return undefined.